src/main/java/org/project36/qualopt/config/social/SocialConfiguration.java

```diff
@@ -95,8 +100,36 @@ public void addConnectionFactories(ConnectionFactoryConfigurer connectionFactory
         }
 
         // Twitter configuration
-        String twitterClientId = environment.getProperty("spring.social.twitter.client-id");
-        String twitterClientSecret = environment.getProperty("spring.social.twitter.client-secret");
+        String twitter_config_file_path = "src\\main\\resources\\config\\.twitter_config.properties";
+        this.configure_Twitter(connectionFactoryConfigurer, twitter_config_file_path);
+
+        // jhipster-needle-add-social-connection-factory
```

Comment:
is this comment still necessary here?

Reply:
It should still be there, as it helps to separate code into subsections.
Same goes for '//Google configuration', '//Facebook configuration'.